### PR TITLE
Merge MichaelDinata-JUnitTests and Update UserGuide

### PR DESCRIPTION
**Pull request by michaeldinata**
Added:
1. JUnit tests for some parser subclasses

Updated:
1. Help command
2. Standardised all logger definitions
3. User Guide

### CODE REVIEW

File: src/test/java/anichan/parser/ViewWatchlistParserTest.java [hunk 1]
```diff
@@ -42,4 +43,12 @@ void parse_nonIntegerField_throwsAniException() {
...
     }
+
+    @Test
+    void parse_tooManyParameters_throwsAniException() {
+        AddToWatchlistParser testParse = new AddToWatchlistParser();
+        assertThrows(AniException.class, () -> {
+            testParse.parse(TOO_MANY_PARAMETERS);
+        });
+    }
 }
```
Comment (ChanJianHao) [inline src/test/java/anichan/parser/ViewWatchlistParserTest.java]: Perhaps add a new line here

File: src/test/java/anichan/command/ViewWatchlistCommandTest.java [hunk 2]
```diff
@@ -66,4 +77,33 @@ void execute_indexLargerThanDataSize_throwsAniException() throws AniException {
...
+        String expectedOutput = expectedOutputBuild.toString();
+
+        ViewWatchlistParser testParser = new ViewWatchlistParser();
+        ViewWatchlistCommand testView = testParser.parse(VALID_WATCHLIST_INDEX);
+        
+        String actualOutput = testView.execute(animeData, storageManager, user);
+        
+        assertEquals(expectedOutput, actualOutput);
+    }
 }
```
Comment (ChanJianHao) [inline src/test/java/anichan/command/ViewWatchlistCommandTest.java]: Would a new line here be good?

File: src/main/java/anichan/parser/InfoParser.java [hunk 3]
```diff
@@ -9,6 +9,7 @@
 
 public class InfoParser extends CommandParser {
     protected static final String ANIME_ID_PARAM = "a";
+    protected static final String TOO_MUCH_ARGUMENTS = "Info command " + TOO_MUCH_FIELDS;
```
Comment (ChanJianHao) [inline src/main/java/anichan/parser/InfoParser.java:12]: Good work with refactoring!

**Review by ChanJianHao — APPROVED** [review]
LGTM 👍🏻

**Review by OngDeZhi — APPROVED** [review]
LGTM :)

**Pull request merged**